about.html

```diff
@@ -0,0 +1,141 @@
+            </div>
+            <div class="para-container-about bg-light">
+                <p class="display-paragrapg">We’re happy to present the draft program schedule for
+                    the 2015 Creative Commons Global Summit in Seoul. In addition to the keynotes,
+                    the program contains a diverse selection of sessions ranging from open business models,
+                    3D printing and design, open education, CC technology, copyright reform advocacy,
+                    open access, and community cooperation. The summit includes Creative Commons partners from around
+                    the world,
+                    and will incorporate sister organizations such as EFF and companies like Shapeways and 500px.
+                    The program highlights several Korean organizations and projects in the creative industries and open
+                    data sector.
+                    Art Centre Nabi, a major art gallery in Korea, is preparing a special exhibition related to CC to
+                    celebrate the Global Summit.</p>
+            </div>
+            <div class="text-center my-3">
+                <p>Please contact us per email for any further question about CC Global Summit 2015</p>
+                <a href="#">chifunwatu@gmail.com</a>
+            </div>
+        </div>
+
+    </section>
+    <section class="mt-5 shadow">
+        <h4 class="text-center my-4">Creative Commons Global Summit 2015 Logo</h4>
+        <p class="mx-4">The winner of the logo contest, which was held from June 8th to July 7th, was decided through online voting and evaluation by the judges.
+            After consultation with the winning designer, the logo of CC Global Summit 2015, which was optimized for use on the website and promotional materials, is as follows.</p>
+            <div class="d-flex justify-content-center">
+                <img src="https://summit.cckorea.org/images/other/about_logo.png" alt="">
+            </div>
+    </section>
+    <section class="mt-5 mb-4 shadow">
+        <h4 class="text-center my-4">View past global summits</h4>
+        <p class="text-center">Take a look at the recent global summits held in Buenos Aires, Argentina and Warsaw, Poland.</p>
+        <div class="row d-flex justify-content-center mx-5 py-4">
+            <div class="col-12 col-md-6 ">
+                <div class="hasHover">
+                    <img src="https://summit.cckorea.org/images/other/about_2013_hover.png" alt="image of global summit 2013" >
+                    <div class="abso"></div>
+                </div>
+                
+            </div>
+            <div class="col-12 col-md-6">
+                <div class="hasHover">
+                    <img src="https://summit.cckorea.org/images/other/about_2011_hover.png" alt="image of global summit 2011" >
+                    <div class="abso"></div>
+                </div>
+               
+            </div>
```

Comment:
- [ ] Please align center the past events cards. In addition, make the background image similar to the original design. 

![image](https://user-images.githubusercontent.com/64894927/133344918-298a2d8d-e53e-4ee0-8369-cdf8fb42dc23.png)
![image](https://user-images.githubusercontent.com/64894927/133344956-b28f4ef1-d6df-43c7-b5ee-00c576c16d5b.png)

Reply:
This issue is well resolved! ✔️